src/poliastro/spacecraft/__init__.py

```diff
@@ -0,0 +1,38 @@
+from astropy import units as u
+
+
+class Spacecraft:
+    """
+    Class to represent a Spacecraft.
+    """
+
+    @u.quantity_input(A=u.km ** 2, C_D=u.one, m=u.kg)
+    def __init__(self, A, C_D, m, **metadata):
+        """
+        Constructor
+
+        Parameters
+        ----------
+        A: ~astropy.units.Quantity
+        Area of the spacecraft (km^2)
+        C_D: ~astropy.units.Quantity
+        Dimensionless drag coefficient ()
+        m: ~astropy.units.Quantity
+        Mass of the Spacecraft (kg)
+        metadata: Dict[object, dict]
+        Optional keyword arguments to Spacecraft
+        """
+
+        self._A = A
+        self._C_D = C_D
+        self._m = m
+        self._metadata = metadata
+        self._A_over_m = A / m
+
+    def get_ballistic_coefficient(self):
+        """Calculates the Ballistic Coefficient (km^2/kg)
+
```

Comment:
Could we include some mathematical description also under the docstring `Notes` section? Just to show current definition also in documentation :rocket: